elyra/cli/pipeline_app.py

```diff
@@ -526,20 +521,41 @@ def describe(json_option, pipeline_path):
     describe_dict["type"] = primary_pipeline.type
     describe_dict["version"] = primary_pipeline.version
     describe_dict["nodes"] = len(primary_pipeline.nodes)
+    describe_dict["scripts"] = set()
+    describe_dict["notebooks"] = set()
     describe_dict["file_dependencies"] = set()
     describe_dict["component_dependencies"] = set()
+    describe_dict[pipeline_constants.MOUNTED_VOLUMES] = set()
+    describe_dict[pipeline_constants.RUNTIME_IMAGE] = set()
     for node in primary_pipeline.nodes:
         # collect information about file dependencies
         for dependency in node.get_component_parameter("dependencies", []):
             describe_dict["file_dependencies"].add(f"{dependency}")
         # collect information about component dependencies
         if node.component_source is not None:
             describe_dict["component_dependencies"].add(node.component_source)
+        # collect information of mounted volumes
+        for mounted_volume in node.get_component_parameter(pipeline_constants.MOUNTED_VOLUMES, []):
+            describe_dict[pipeline_constants.MOUNTED_VOLUMES].add(f"{mounted_volume.split('=')[-1]}")
+        # collection runtime image details
+        describe_dict[pipeline_constants.RUNTIME_IMAGE].add(
+            node.get_component_parameter(pipeline_constants.RUNTIME_IMAGE, "")
+        )
+        # collect notebook / script name when pipeline is generic
+        if describe_dict["type"] is None:
+            temp_value = Path(node.get_component_parameter("filename", "")).name
+            if Path(temp_value).suffix == ".ipynb":
+                describe_dict["notebooks"].add(temp_value)
+            else:
+                describe_dict["scripts"].add(temp_value)
 
     if not json_option:
-        for key in pipeline_keys:
+        click.echo()
+
+        print_banner("Elyra Pipeline details")
+        for key in list(describe_dict.keys()):
             readable_key = " ".join(key.title().split("_"))
-            if key in iter_keys:
+            if type(describe_dict[key]) is set:
```

Comment:
Nit - slightly better: 
```suggestion
            if isinstance(describe_dict[key], set):
```

Reply:
Done